src/main/java/ru/yandex/practicum/filmorate/model/Director.java

```diff
@@ -0,0 +1,15 @@
+package ru.yandex.practicum.filmorate.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Director {
+    private Integer id;
+    private String name;
```

Comment:
Предлагаю валидацию делегировать аннотации.

```suggestion
    @NotBlank(message = "Director name must be completed")
    private String name;
```